Perlang.Interpreter/Resolver.cs

```diff
@@ -280,7 +292,8 @@ public VoidObject VisitVarStmt(Stmt.Var stmt)
                 Resolve(stmt.Initializer);
             }
 
-            Define(stmt.Name);
+            // TODO: Get the type reference from the initializer instead.
+            Define(stmt.Name, TypeReference.None);
```

Comment:
(Again, the necessary preconditions for putting a real `TypeReference` value in place here isn't really there yet.)